src/js/components/tablero/GraphCard.jsx

```diff
@@ -17,9 +17,12 @@ const GraphCard = SortableElement(({
   const cardData = data;
   let graph;
   let filter = 0;
+  let label = 'Last';
```

Comment:
What's the point of those changes?

Reply:
This is for the time filter label to change between "Last x months" for indicators that show past data and "Next x months" for the Expiration Summary